### ISSUE TITLE
Cannot run on Mac / Python 2.7 after install with pip

### ISSUE THREAD

**Issue by adedov** [blabla1337/skf-flask#104]
Hi!

I installed owasp-skf with pip as mentioned in documentation but it does not start:
```
% pip show owasp-skf
Name: owasp-skf
Version: 1.3.21
Summary: The OWASP Security Knowledge Framework
Home-page: https://github.com/blabla1337/skf-flask
Author: Glenn ten Cate, Riccardo ten Cate
Author-email: gtencate@schubergphilis.com, r.tencate77@gmail.com
License: AGPLV3
Location: /usr/local/lib/python2.7/site-packages
Requires: markdown, BeautifulSoup, python-docx, lxml, cryptography, pyOpenSSL, requests, importlib, Bcrypt, flask-bcrypt
```

```
% pwd
/usr/local/lib/python2.7/site-packages/skf

% python skf.py
Traceback (most recent call last):
  File "skf.py", line 30, in <module>
    from flask_bcrypt.bcrypt import Bcrypt
ImportError: No module named bcrypt
```

It looks like it relies on different API version from Flask?

```
% python
Python 2.7.10 (default, Jul 30 2016, 18:31:42)
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.34)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from flask_bcrypt.bcrypt import Bcrypt
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named bcrypt

>>> from flask_bcrypt import Bcrypt
>>>
```

**Comment by RiieCco** [comment]
Hey Anton,

We do indeed have some dependency issues which we did not took into
consideration while building the pip files.

There is also a docker container available in which to run skf in very
easily.

Docker for mac is available here:
https://docs.docker.com/docker-for-mac/


When Docker is available, the fastest way to start using the SKF project is
using the pre-built container hosted at Docker hub.

docker run -ti -p 127.0.0.1:443:5443 blabla1337/skf-flask

The application will greet you on: https://127.0.0.1

This container always has the very latest version from the repository.


I hope this helps!


With kind regards,

SKF team!

2016-11-18 6:42 GMT+01:00 Anton Dedov <notifications@github.com>:

**Comment by adedov** [comment]
Thanks!

It does work, but reports warning about bcrypt:

```
skf.py:30: ExtDeprecationWarning: Importing flask.ext.bcrypt is deprecated, use flask_bcrypt instead.
  from flask.ext.bcrypt import Bcrypt
```

**Comment by adedov** [comment]
Do you have instruction on how to configure docker image to save state between restarts? Should I make application store skf.db on a Docker volume?

**Comment by RiieCco** [comment]
Yeah as far as i know you can mount/map docker files to local files.
I did not tested it yet with SKF but for instance when i use jenkins i
start the docker image as following:

docker run -p 8080:8080 -p 50000:50000 -v
/Users/riiecco/docker_vol:/var/jenkins_home jenkins

Maybe that helps :-)

I can test for you on a later time, but this should be the docker way of
saving state as far as i understood :-)

2016-11-18 22:44 GMT+01:00 Anton Dedov <notifications@github.com>:

**Comment by adedov** [comment]
Thanks! Let me investigate and write you back later.

**Comment by adedov** [comment]
Investigated, created new issue :)
https://github.com/blabla1337/skf-flask/issues/105

**Comment by blabla1337** [comment]
Fixed in new release
